.chloggen/azuremonitor-receiver-semconv.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
I know this receiver is in development, but worth pointing out this should be marked as a breaking change since you're adding new default resource attributes enabled by default.

Reply:
This makes sense to me. Thanks for noting.